.github/workflows/deployment_gke_dev.yaml

```diff
@@ -12,14 +12,14 @@ on:
   workflow_dispatch:
 
 env:
-  PROJECT_ID: solutions-template-sandbox
-  SKAFFOLD_DEFAULT_REPO: gcr.io/solutions-template-sandbox
+  PROJECT_ID: solutions-template-e2etest
+  SKAFFOLD_DEFAULT_REPO: gcr.io/solutions-template-e2etest
   SKAFFOLD_NAMESPACE: default
   GKE_CLUSTER: main-cluster
   GKE_REGION: us-central1
   KUSTOMIZE_VERSION: 4.1.3
   SKAFFOLD_VERSION: 2.0.3
```

Comment:
Are Skaffold and Kustomize versions required?

Reply:
Yes, it's highly recommended here. Especially the specific Skaffold version that we tested locally.

Reply:
We should externalize these versions in one place and replace with environment variables

Reply:
Can be done later

Reply:
It would need a significant refactor of the GHA as the way to manage the versions. Let's brainstorm alternative approaches later.